mcs/class/monodoc/Monodoc.Ecma/EcmaDesc.cs

```diff
@@ -235,17 +235,21 @@ void ConstructCRef (StringBuilder sb)
 			sb.Append ('.');
 			sb.Append (TypeName);
 			if (GenericTypeArguments != null) {
```

Comment:
As mentioned in another commit, most of this section is simply due to re-indenting code. The patch would be much smaller, and thus would impact history less, if you moved the `GenericTypeArgumentsIsNumeric` check to be a "sibling" of this branch:

    if (GenericTypeArguments != null && GenericTypeArgumentsIsNumeric) {
        sb.AppendFormat ("`{0}", GenericTypeArgumentsCount);
    } else if (GenericTypeArguments != null) {
        ...unchanged from before...
    }